addons/sale/models/account_move.py

```diff
@@ -232,3 +232,7 @@ def _sale_get_invoice_price(self, order):
         if currency_id and currency_id != order.currency_id:
             price_unit = currency_id._convert(price_unit, order.currency_id, order.company_id, order.date_order or fields.Date.today())
         return price_unit
+
+    def _get_downpayment_lines(self):
+        # OVERRIDE
+        return self.env['account.move.line'].search([('sale_line_ids', 'in', self.sale_line_ids.filtered('is_downpayment').ids)]).filtered(lambda line: line.move_id._is_downpayment())
```

Comment:
why the lines and not the moves themselves?  @dbkosky

Reply:
It's more pertinent to the information required in the xml in l10n_it_edi. 
The Descrizione element is per-line.

Reply:
but normally we will only have one line per invoice.